crowd_sim/envs/crowd_sim.py

```diff
@@ -314,6 +385,34 @@ def reset(self, phase='test', test_case=None):
     def onestep_lookahead(self, action):
         return self.step(action, update=False)
 
+    def generate_valid_goal(self, gx, gy, r):
+        while True:
+            collide = False
+            for agent in [self.robot] + self.humans + self.obs:
+                if norm((gx - agent.gx, gy - agent.gy)) < r + agent.radius + self.discomfort_dist:
+                    collide = True
+                    break
+            if not collide:
+                break
+        return gx, gy
```

Comment:
I think we can remove this, since this function is not being used.

Reply:
I would like to keep it at the moment because it might have some uses later I think